docs/jax.numpy.rst

```diff
@@ -474,6 +474,30 @@ instantiate :class:`DeviceArray` objects manually, but rather will create them v
 :mod:`jax.numpy` functions like :func:`~jax.numpy.array`, :func:`~jax.numpy.arange`,
 :func:`~jax.numpy.linspace`, and others listed above.
 
+Copying and Serialization
+~~~~~~~~~~~~~~~~~~~~~~~~~
+DeviceArray objects are designed to work seamlessly with Python standard library
+tools where appropriate.
+
+With the built-in :mod:`copy` module, when :func:`copy.copy` or :func:`copy.deepcopy`
+encounder a :class:`~jax.numpy.DeviceArray`, it is equivalent to calling the
+:meth:`~jaxlib.xla_extension.DeviceArray.copy` method, which will create a copy of
```

Comment:
A note, not necessarily for this change: I don't consider anything in `jaxlib` to be a public API. We should have public `jax.` names for anything we need to refer to.

Reply:
Yeah, unfortunately array methods are only defined on the xla extension version, so we can only cross-reference them that way: https://jax.readthedocs.io/en/latest/jax.numpy.html#jax.numpy.DeviceArray